vmdb/app/controllers/miq_ae_customization_controller/dialogs.rb

```diff
@@ -194,6 +194,7 @@ def dialog_edit
     when 'cancel'
       @edit = session[:edit] = nil # clean out the saved info
       self.x_active_tree = :dialogs_tree
+      @record = identify_record(params[:id], Dialog) if params[:id]
```

Comment:
@h-kataria Code requires @record to be set when cancelling out of edit mode. I think we should leave code fix as is.